kiosk/src/Components/HighScoreInitial.tsx

```diff
@@ -38,14 +38,14 @@ const HighScoreInitial: React.FC<IProps> = ({
 
     // Handle DPadUp button press
     useOnControlPress(
-        [onCharacterChanged, isSelected, index],
+        [onCharacterChanged, isSelected, setIndex],
```

Comment:
I don't see how this change will fix the double skip issue. Wouldn't we want the `useEffect` dependency to be `index`? Is it possible that it's skipping two because of the fact that in `Strict` mode, React will run `useEffect` twice?

Reply:
I see your point about React strict mode though. In this case it's fine. The underlying useEffect can be called many times without issue. It has no side effects.

Reply:
Ah, that makes sense. So should we take `setIndex` out or is it a good reference to keep in to say "this will cause the index to change"?

Reply:
Because it is referenced, I'd prefer to keep it in. I think an eslint exhaustive deps check would complain otherwise. I'll take a todo here is to figure out why the inclusion of index is causing side effects.